README.md

```diff
@@ -149,6 +149,18 @@ schedule_task(local_time)  # ✗ Type error
 
 We welcome contributions! See [CONTRIBUTING.md](https://github.com/channable/heliclockter/blob/master/CONTRIBUTING.md).
 
+## Python and Pydantic compatibility
+
+The table below shows which Pydantic and Python versions are supported for which heliclockter version.
+Note that the latest version of heliclockter dropped support for Pydantic v1 code completely, meaning that you
+also can't use heliclockter in combination with the `pydantic.v1` module in Pydantic v2.
+
+| heliclockter version | Pydantic support | Python support |
+| -- | -- | -- |
+| 1.* | v1, v2 | 3.9, 3.10, 3.11, 3.12, 3.13 |
+| 2.* | v1, v2 | 3.10, 3.11, 3.12, 3.13 |
+| 3.* | v2 | 3.10, 3.11, 3.12, 3.13, 3.14 |
```

Comment:
Nice 👍🏼

Reply:
Actually, small nit. We only started supporting pydantic v2 as of heliclockter v 1.1.0, the table should probably reflect that